components/nav.tsx

```diff
@@ -55,40 +55,40 @@ export default function Nav({}) {
         }`}
       >
         <div className="text-md font-semibold	pt-2 lg:pt-0 lg:flex-grow text-red-700">
-          <s className="mr-4">FEEL the love</s>
           <a
-            href={`https://twitter.com/intent/tweet?text=${encodeURIComponent(
-              // @TODO: link to the user's "gifted" page to show off -- "I just gifted, you should too"
-              "SHARE the Rage! https://ragingsantas.xyz @RagingSantasNFT"
-            )}`}
-            target="_blank"
+            href="#mint"
             className="block mt-4 lg:inline-block lg:mt-0 text-teal-200 hover:text-white mr-4 cursor-pointer"
           >
-            SHARE the Rage
+            GIVE
           </a>
           <a
-            href="#mint"
+            href="#reveal"
             className="block mt-4 lg:inline-block lg:mt-0 text-teal-200 hover:text-white mr-4 cursor-pointer"
           >
-            GIVE for Santa
+            CLAIM
           </a>
           <a
             href="#faq"
             className="block mt-4 lg:inline-block lg:mt-0 text-teal-200 hover:text-white mr-4 cursor-pointer"
           >
-            ASK a question
+            FAQ
           </a>
           <a
-            href="#reveal"
+            href="#the-santas"
             className="block mt-4 lg:inline-block lg:mt-0 text-teal-200 hover:text-white mr-4 cursor-pointer"
           >
-            OPEN presents
+            TEAM
           </a>
+          <s className="mr-4">FEEL the love</s>
```

Comment:
I'd say we just drop this one 🤷‍♀️

Reply:
ok! yeah we're sorta dropping the snarky old-timey ad-speak so i will remove